src/components/Dialogs/CreateWorkspaceDialogContainer.tsx

```diff
@@ -32,10 +23,14 @@ export function CreateWorkspaceDialogContainer({
   isOpen,
   setIsOpen,
   project = '',
+  useCreateWorkspace: useCreateWorkspaceHook = useCreateWorkspace,
+  useAuthOnboarding: useAuthOnboardingHook = useAuthOnboarding,
 }: {
   isOpen: boolean;
   setIsOpen: (isOpen: boolean) => void;
   project?: string;
+  useCreateWorkspace?: typeof useCreateWorkspace;
```

Comment:
Let's stick to the naming convention of importing the hooks with the underscore `_` prefix, so it’s the same pattern everywhere. Check out `CreateManagedControlPlaneWizardContainer` for reference.